db/column_family.cc

```diff
@@ -601,6 +625,16 @@ ColumnFamilyData::~ColumnFamilyData() {
   for (MemTable* m : to_delete) {
     delete m;
   }
+
+  if (db_paths_registered_) {
+    Status s = ioptions_.env->UnregisterDbPaths(GetDbPaths());
```

Comment:
Same here. It will be ideal to call FileSystem rather than Env, as it is the new API.

Reply:
Some tests rely on `EnvWrapper` right now, using `fs` here would break those tests.

Also, I find that most of the code now still uses `env` as a public API, `FileSystem` is more like a subset or implementation detail of `Env`.

Reply:
left a TODO in the code